web/src/main/java/se/inera/intyg/webcert/web/service/facade/list/previous/CertificateForPatientServiceImpl.java

```diff
@@ -71,9 +74,16 @@ private List<ListIntygEntry> getCertificatesFromWC(Personnummer patientId, List<
         LOG.debug("UtkastService returned {} certificates", certificatesFromWC.size());
         return certificatesFromWC.stream()
             .map(IntygDraftsConverter::convertUtkastToListIntygEntry)
+            .map(this::setRelations)
             .collect(Collectors.toList());
     }
 
+    private ListIntygEntry setRelations(ListIntygEntry entry) {
+        final var relations = certificateRelationService.getRelations(entry.getIntygId());
```

Comment:
Ideally this would get done in IntygDraftsConvert, but thats not possible since the method there is static and hinders DI.

Reply:
This could possibly have a negative impact on performance. I would like to analyse this a bit deeper before I can finalize a review.